plugin/trino-druid/src/test/java/io/trino/plugin/druid/BaseDruidConnectorTest.java

```diff
@@ -323,4 +325,73 @@ public void testNativeQueryInsertStatementTableExists()
     {
         throw new SkipException("cannot create test table for Druid");
     }
+
+    @Test
+    public void testPredicatePushdown()
```

Comment:
Do we need such test in `BaseJdbcConnectorTest`?

Reply:
I think @kokosing meant that we should have some predicate pushdown tests in BaseJdbcConnectorTest (or BCT perhaps?). That should be a separate effort however. Druid specific ones can be added as separate methods.